81.md

```diff
@@ -0,0 +1,73 @@
+NIP-81
+======
+
+Relationship Status
+-------------------
+
+`draft` `optional`
+
+This NIP defines two new event kinds (`30382` and `30383`) to document the relationship between two keys. 
+
+Both kinds offer public and private tags to describe that relationship. Private tags are JSON Stringified, [NIP-44](44.md)-encrypted to the signer's keys and placed inside the `.content` of the event. 
+
+### Public Relationship Status
+
+Event kind `30382` is used when the relationship itself is public, with the `d` tag receiving the public key each target key.
+
+```js
+{
+  "kind": 30382,
+  "tags": [
+    ["d", "e88a691e98d9987c964521dff60025f60700378a4879180dcbbb4a5027850411"],
+    ["n", "Follows"],
+    ["n", "Bitcoiners"],
```

Comment:
```suggestion
    ["n", "follow"],
    ["n", "bitcoiner"],
```

Reply:
Can we convention unbound list names to using lowercase, singular form, English words separated by dash?
Added this info to the the unbound list NIP.